clean.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+rm config_parser
```

Comment:
I respectfully disagree: it's apparent from the filenames + build scripts that these correspond to the binaries produced, and dSYM files are [standard container files for debugging symbols](https://stackoverflow.com/questions/3656391/whats-the-dsym-and-how-to-use-it-ios-sdk) across all Macs.